doc/version_history.rst

```diff
@@ -6,6 +6,11 @@
 Version History
 ===============
 
+v1.21.0
```

Comment:
We have not release 1.20 yet. I am waiting to merge all the new Scripts to do that in a single release. Can you please add your changes of version 1.20 below?

Reply:
Done